### ISSUE TITLE
cli.confirm should allow for defaults

### ISSUE THREAD

**Issue by TranquilMarmot** [oclif/cli-ux#134]
It's common in CLIs to have a yes/no prompt with a default, i.e. `(Y/n)` implies that just pressing enter will default to "yes".

It would be nice to have an option for `cli.confirm` to be able to accept a default answer (this would probably also be nice for `cli.prompt`)

**Comment by RasPhilCo** [comment]
Good idea! Should be a straightforward PR if someone wants to pick this up.

**Comment by youyiqin** [comment]
we need it.
